pineappl_py/README.md

```diff
@@ -12,3 +12,49 @@ For installation instructions see the [documentation].
 [PyO3]: https://pyo3.rs
 [Rust API]: https://docs.rs/pineappl
 [documentation]: https://pineappl.readthedocs.io/en/latest/installation.html
+
+## Development
+
+Run
+
+```shell
+python -m venv env && . env/bin/activate
+```
+
+to setup a new environment and check that `pip --version` returns at least `pip
+22.0 from ...`. If not, upgrade `pip` via
+
+```shell
+pip install -U pip
+```
+
+Next, install `maturin`:
+
+```shell
+pip install maturin
+```
+
+Run
+
+```shell
+maturin develop
+```
+
+to build the project, which also installs it into the environment so that it
+can be used in Python projects that use the same environment.
+
+### Documentation
+
+Run the following once to install the documentation's dependencies:
+
+```shell
+pip install .[docs]
```

Comment:
I'm not sure whether I'd recommend single quotes:
```suggestion
pip install '.[docs]'
```

I know that it is not required with Bash and Nu, but it is required with Zsh, which is the default on MacOS.
Of course, I hope people using Zsh are already aware of it. But when it's a combination of the default shell and a non-ubiquitous feature (extras), you know someone may experience the issue. Though, it could be easily solved.

The single quotes instead are always working in my experience (despite being less elegant and a bit more annoying...).